PublicFolders/ValidateEXOPFDumpster.ps1

```diff
@@ -346,7 +382,7 @@ function ValidateDumpsterChildren {
         WriteToScreenAndLog -Issue $Issue -Fix $Fix
     }
 }
-function ValidateMEPfGuid {
+function ValidateMePfGuid {
```

Comment:
You should probably comment here that `Me` stands for Mail Enabled public folder. This makes it easier for a reader of the code to understand what this is.

Reply:
Done.

Reply:
push the changes so i can see them.

Reply:
Sure on it.